lib/bolt/config.rb

```diff
@@ -567,7 +568,18 @@ def plugins
     end
 
     def plugin_hooks
-      @data['plugin_hooks']
+      if @data['plugin-hooks'].any?
```

Comment:
I think this might be easier to follow as a single-level if statement:
```
if @data['plugin-hooks'].any? && @data['plugin_hooks'].any?
  issue warning and return plugin-hooks
elsif @data['plugin-hooks'].any?
  @data['plugin-hooks']
else
  @data['plugin_hooks']
end
```
```